.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+/*eslint linebreak-style: ["error", "windows"]*/
```

Comment:
Kindly remove this line of code. Endeavour to avoid modifying the linters configuration files